README.md

```diff
@@ -45,6 +45,25 @@ Then, run the development server:
 npm run dev
 ```
 
+### Server side
+#### Pre-requisites
+Please ensure you have:
+- [.NET sdk](https://dotnet.microsoft.com/download)
+
+In order for favourites feature to work you must run the server api.
+
+First install the dependencies for the project:
+```bash
+cd server
+dotnet build
+```
+
+Then, run the development server:
+```bash
+cd server/api
+dotnet run
+```
+
 ### GitHub Authentication
 
 To get the GitHub authentication to work, you need to create a `.env` file in the `gitdash` directory with the following content
```

Comment:
Can you rename `gitdash` here to `client`?

Reply:
Done. In my latest commit

Reply:
But the file review section still shows `gitdash` and not `client` in the README.md

Reply:
Can you change `gitdash` on line 69 to `client`?

Reply:
Oh my bad, I thought you were referring the folder. On it now